### ISSUE TITLE
Assassin's Creed Valhalla Crash

### ISSUE THREAD

**Issue by GigaEgo** [tuffee88/d3d12ProxyEdrDx11_0#24]
While I can confirm that the game does boot and play at a steady fps it will eventually quickly crash.
I've tested this on my GTX 950M but have also confirmed with others that it crashes quite frequently. for me within minutes or even seconds after I got the shield.
For some lowering the games resolution and scaling in game to 720 at 50% gives them a bit more time but i cannot confirm this at the moment.

**Comment by tuffee88** [comment]
Sorry to hear that, I take it you tried all the usual stuff (windows + driver updates, disable overlays and overclocking, ecc.) ?

Please also try to run this with vkd3d-proton (see README).

**Comment by mojmass** [comment]
I used vkd3d-proton and your fix to try to run Assassin's Creed Valhalla
Without vkd3d, the game runs very good but as mentioned above it crashes at certain points.
When the game starts, before the disclaimer message, there is a brief moment where it shows a white box (like it's loading what's to come) 
when I use vkd3d-proton the games gets stuck on this white box screen and stays there, but it is not frozen or in a not responding state (according to task manager)

**Comment by GigaEgo** [comment]
haven't tried overclocking, Bit scared to with this rickety old graphics card (plus don't really know the inner workings of overclocking either lol).

**Comment by GigaEgo** [comment]
after overclocking I figured out it seems to have changed nothing

**Comment by tuffee88** [comment]
I'll leave this Issue pinned so it's easy to find: 
AC Valhalla crashes frequently (depending on settings won't even get to the intro sequence on Kepler/Maxwell 1st Gen GPUs). 

Using vkd3d-proton (tested v2.0 - 2.6) doesn't help either (game won't boot).

-> Sorry, I won't be able to fix this (out of the scope of this project), comment added to README for other players though.

**Comment by GigaEgo** [comment]
That's fair, its a bit upsetting but so is life lol, I got elden ring to occupy myself anyhow :)))

**Comment by silenthunter93** [comment]
Yeah AC Valhalla is crashing for me as well, before this mod I wasn;t even able to launch the game since my gpu doesn't fully support DX12. With this DX mod I was able to launch the game and start as well but it crashes near the end of the beginning cut-scene. I was also able to launch Elden Ring with this mod but tested only up until char creation, before this mod it always crashed with white screen.

**PC Specs:**
CPU: Xeon E5-2650V2 x2 (16 Cores - 32 Logical)
RAM: 48GB
GPU: Quadro K5000 x2 SLI - 8GB

**Comment by Sitis133** [comment]
[tuffee88](https://github.com/tuffee88)
maybe the problem is the lack of support for AVX instructions ?

**Comment by tuffee88** [comment]
> [tuffee88](https://github.com/tuffee88) maybe the problem is the lack of support for AVX instructions ?

Sadly not - the exact same issue (CDT shortly after starting the game, time depends on graphics settings) happens with much newer CPUs that have the latest AVX/SSE, etc. instruction sets available.

**Comment by Gaming1992** [comment]
> I used vkd3d-proton and your fix to try to run Assassin's Creed Valhalla Without vkd3d, the game runs very good but as mentioned above it crashes at certain points. When the game starts, before the disclaimer message, there is a brief moment where it shows a white box (like it's loading what's to come) when I use vkd3d-proton the games gets stuck on this white box screen and stays there, but it is not frozen or in a not responding state (according to task manager)

How to use vkd3d Proton?

**Comment by HaddadCSharp** [comment]
> While I can confirm that the game does boot and play at a steady fps it will eventually quickly crash. I've tested this on my GTX 950M but have also confirmed with others that it crashes quite frequently. for me within minutes or even seconds after I got the shield. For some lowering the games resolution and scaling in game to 720 at 50% gives them a bit more time but i cannot confirm this at the moment.

I have exactly the same problem.
VGA: NVIDIA GTX 960m (Laptop)

**Comment by Grey786** [comment]
> I'll leave this Issue pinned so it's easy to find: AC Valhalla crashes frequently (depending on settings won't even get to the intro sequence on Kepler/Maxwell 1st Gen GPUs).
> 
> Using vkd3d-proton (tested v2.0 - 2.6) doesn't help either (game won't boot).
> 
> -> Sorry, I won't be able to fix this (out of the scope of this project), comment added to README for other players though.

Listen dude can you fix the crashing issue. It runs fine until it crashes. I mean the solution is near, just have to make the crashing go away. Plus this fix should be made avalaible just for AC valallah. For those gpu's that do not have the directx feature level crap but have the ability to run the game.
